scripts/SREP/retry-pruning-cronjob/script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Delete jobs that fail in openshift-sre-pruning namespace.
+
+set -x
+set -e
+set -o nounset
+set -o pipefail
+
+readonly PRUNING_NS=openshift-sre-pruning
+
+Builds_Pruner=()
+Deployements_Pruner=()
+FailedJobs=()
```

Comment:
Can we set the variable as all upper cases? like `FAILED_JOBS`

Reply:
Sure!